samples/DotNet/Azure.Messaging.EventHubs/AnomalyDetector/Producer/Program.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Producer
+{
+    using System;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Azure.Messaging.EventHubs;
+    using System.Collections.Generic;
+    using System.IO;
+
+    public class Program
+    {        
+        private const string EventHubConnectionString = "<EVENT HUBS NAMESPACE CONNECTION STRING>";
+        private const string EventHubName = "<EVENT HUB NAME>";
+        private const string TransactionsDumpFile = "mocktransactions.csv";
+
+        private static EventHubProducerClient producerClient;
+
+        public static int Main()
```

Comment:
You may want to consider merging `Main` and `MainAsync`, as async entry points are supported in your target framework.

Reply:
Done.